File: src/codes/Users_Manual/segment/segment_change_transition.py
~from pyautd3 import AUTD3, Controller, Segment, SwapSegment, TransitionMode
~from pyautd3.link.nop import Nop
~autd = Controller.open([AUTD3()], Nop())
autd.send(SwapSegment.Modulation(Segment.S1, TransitionMode.Immediate))

File: src/codes/Users_Manual/segment/segment_change_transition.rs
# use autd3::prelude::*;
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let mut autd = Controller::open([AUTD3::default()], autd3::link::Nop::new())?;
autd.send(SwapSegment::Modulation(Segment::S1, TransitionMode::Immediate))?;
# Ok(())
# }